source/horoscope/pages/daily.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="en">
+   <head>
+      <meta charset="UTF-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <title>Horoscope App, Team #5</title>
+      <link rel="icon" type="image/x-icon" href="../images/star_favicon.ico">
+      <link rel="stylesheet" type="text/css" href="../css/daily.css">
+      <script src="../js/daily.js" type="module"></script>
+   </head>
+<body>
+   <main>
+      <header>
+         <h1>Daily Horoscopes</h1>
+         <nav>
+            <button id="back">Go Back to Main Page</button>
+         </nav>
+      </header>
+      <div>
```

Comment:
small nitpick: maybe instead of div, you can use the Section tag to make it more semantic.